### PR TITLE
[5.2] Fix updateRememberToken call during logout

### PR DESCRIPTION
**Pull request by andrejbaran**
During logout call updateRememberToken with $token=null as stated in Authentication documentation:

> The updateRememberToken method updates the $user field remember_token with the new $token. The new token can be either a fresh token, assigned on a successful "remember me" login attempt, or a null when the user is logged out.

### CODE REVIEW

File: src/Illuminate/Auth/Guard.php [hunk 1]
```diff
@@ -605,6 +603,20 @@ protected function createRememberTokenIfDoesntExist(UserContract $user)
     }
 
     /**
+     * Discards remember token for the user.
+     * @param  \Illuminate\Contracts\Auth\Authenticatable  $user
```
Comment (GrahamCampbell) [inline src/Illuminate/Auth/Guard.php]: cs

**Comment by andrejbaran** [comment]
Hey @GrahamCampbell should be ok now.

**Comment by taylorotwell** [comment]
We should just fix the documentation.

**Comment by andrejbaran** [comment]
Hey @taylorotwell would you care to elaborate a bit?

**Pull request closed without merging**